consensus/ibft/ibft.go

```diff
@@ -541,7 +541,7 @@ func (i *Ibft) runSyncState() {
 			isValidator = i.isValidSnapshot()
 
 			return isValidator
-		})
+		}, i.blockTime)
```

Comment:
If `--block-time 0` is given, there will be no timeout?

Reply:
Not something we ever considered to be set to 0. (afaik, if it is, other parts of the system start acting weird)

Do you think there should be an exception given a 0 block time ?

Reply:
I tried to give 0 to `--block-time`. Then IBFT has 0 in blockTime. After that, syncer does not check this value